pyfr/backends/openmp/types.py

```diff
@@ -25,10 +27,38 @@ def onalloc(self, basedata, offset):
         del self._initval
 
     def _get(self):
-        return self._unpack(self.data[:, :self.ncol])
+        return self._unpack(self.data[:, :, :])
 
     def _set(self, ary):
-        self.data[:, :self.ncol] = self._pack(ary)
+        self.data[:, :, :] = self._pack(ary)
+
+    def _pack(self, ary):
```

Comment:
Can this not be integrated with the parent class?  Currently it is weird that the high level base class knows something about blocking, but not everything.